Core/Src/compute.c

```diff
@@ -535,6 +535,30 @@ void compute_send_fault_message(uint8_t status, int16_t curr, int16_t in_dcl)
 	queue_can_msg(bms_can_msgs[FAULT]);
 }
 
+void compute_send_fault_timer_message(uint8_t start_stop, uint16_t fault_code,
+				      uint16_t data_1)
+{
+	struct __attribute__((__packed__)) {
+		uint8_t start_stop;
+		uint16_t fault_code;
+		int16_t data_1;
+	} fault_timer_msg_data;
+
+	fault_timer_msg_data.start_stop = start_stop;
+	fault_timer_msg_data.fault_code = fault_code;
```

Comment:
I know this sounds wierd, but we need to compress the faults down to fit.  A little context: Since there can be multiple faults at once, we use 32 bits to represent all faults in our status messages.  Each bit represents if a fault is active or not.  
However, you are sending the singular fault I believe (feel free to look to confirm).  This means we do not need this 32 bit integer, and we can **log_2** data in order to compress it for sending inside of this message.  Then a 16 bit integer does indeed work. Feel free to let me know if this makes sense, but the TLDR is to fit the fault ID into the variable, just log base 2 it.